src/h3/_polygon.py

```diff
@@ -118,97 +166,99 @@ def __geo_interface__(self):
 """
 
 
-if True:  # functions below should be inverses of each other
-    def _mpoly_to_LL3(mpoly):
-        ll3 = [
-            _polygon_to_LL2(poly)
-            for poly in mpoly
-        ]
+def _mpoly_to_LL3(mpoly):
+    ll3 = [
+        _polygon_to_LL2(poly)
+        for poly in mpoly
+    ]
 
-        return ll3
+    return ll3
 
-    def _LL3_to_mpoly(ll3):
-        polys = [
-            _LL2_to_polygon(ll2)
-            for ll2 in ll3
-        ]
 
-        mpoly = H3MultiPoly(*polys)
+def _LL3_to_mpoly(ll3):
+    polys = [
+        _LL2_to_polygon(ll2)
+        for ll2 in ll3
+    ]
 
-        return mpoly
+    mpoly = H3MultiPoly(*polys)
 
+    return mpoly
 
-if True:  # functions below should be inverses of each other
-    def _polygon_to_LL2(h3poly):
-        ll2 = [h3poly.outer] + list(h3poly.holes)
-        ll2 = [
-            _close_ring(_swap_latlng(ll1))
-            for ll1 in ll2
-        ]
 
-        return ll2
+# functions below should be inverses of each other
+def _polygon_to_LL2(h3poly):
+    ll2 = [h3poly.outer] + list(h3poly.holes)
+    ll2 = [
+        _close_ring(_swap_latlng(ll1))
+        for ll1 in ll2
+    ]
 
-    def _LL2_to_polygon(ll2):
-        ll2 = [
-            _swap_latlng(ll1)
-            for ll1 in ll2
-        ]
-        h3poly = H3Poly(*ll2)
+    return ll2
 
-        return h3poly
 
+def _LL2_to_polygon(ll2):
+    ll2 = [
+        _swap_latlng(ll1)
+        for ll1 in ll2
+    ]
```

Comment:
Agreed that it would be nice to make these symmetric. I'd do that in the `shape_to_geo` function as described here: https://github.com/uber/h3-py/pull/322#issuecomment-1673880758